packages/components/src/query-controls/README.md

```diff
@@ -32,19 +32,29 @@ const QUERY_DEFAULTS = {
 	numberOfItems: 10,
 	order: 'asc',
 	orderBy: 'title',
+	orderByOptions: [
+		{
+			label: 'Title Ascending',
+			value: 'title/asc',
+		},
+		{
+			label: 'Title Descending',
+			value: 'title/desc',
+		},
+	],
 };
 
 const MyQueryControls = () => {
 	const [ query, setQuery ] = useState( QUERY_DEFAULTS );
-	const { category, categories, maxItems, minItems, numberOfItems, order, orderBy  } = query;
+	const { category, categories, maxItems, minItems, numberOfItems, order, orderBy, orderByOptions } = query;
```

Comment:
I think this is incorrect. The `orderByOptions` value isn't stored in the `query` attribute; it is handled by `order` and `orderBy`.

The `orderByOptions` is just a value you can pass to `QueryControls`. It can be a static array or dynamic like `useOrderByOptions`.

Reply:
I've removed it for now. Please see my comment here for further discussion https://github.com/WordPress/gutenberg/pull/68781#discussion_r1975060711